### DISCUSSION TITLE
Status of Qiqqa Development

### DISCUSSION THREAD

**Discussion by Patches-weaver** [jimmejardine/qiqqa-open-source#385] [Q&A]
I love Qiqqa.  It is such a great program and such a great idea.  It has so much potential to become even more than it is now.  

Is Qiqqa still being actively developed?

**Comment by klmckinney** [comment]
Hi TeacherBrian,

Ger Hobbelt has done an excellent job of repository maintenance and it is possible to build and explore the codebase.
I have been doing just that, in my limited free time.
There are a number  of software technologyies in use and I find it hard to "get a bearing".
Mostly, I work with the existing code and do direct manipulation of the SQLite database to meet my needs.
I plan to do more direct Qiqqa development in the future,  I just don't know when.

-Kenelm
